### ISSUE TITLE
Broken on Firefox Nightly 71.0a1 (2019-09-03) (64-bit)

### ISSUE THREAD

**Issue by JohnLGalt** [projectdelphai/panorama-tab-groups#144]
Been using this addon for a while on Nightly builds.  All of a sudden, today, when I use the group button to load a different group, it is not hiding the previous group I was viewing, so all of the tabs are starting to get jumbled together.

Windows 10 Pro x64 1903
Firefox Nightly 71.0a1 (2019-09-03) (64-bit)
PTG 0.8.9

If there is a trace or anything else I can grab, please let me know.

**Comment by projectdelphai** [comment]
Please send all requests and bugs for Panorama Tab Groups to the Github
issues page <https://github.com/projectdelphai/panorama-tab-groups/issues>.
If you email me personally, it's harder to keep track of requests and
remember to respond.

If you don't have a Github account and are unwilling to make one to leave a
response, please let me know. I'll try and respond when I can.

Thanks,
Reuben Castelino

**Comment by vaxul** [comment]
Oh, using an alpha version is quite risky 😅

> If there is a trace or anything else I can grab, please let me know. 

I'm not sure what you like to do. You want to to drag the groups?

**Comment by projectdelphai** [comment]
Ignore my post, thought that was a personal email not a github email 😭

**Comment by projectdelphai** [comment]
Unfortunately we don't have very good logging set up right now so there's no backtrace you can grab for us.

**Comment by JohnLGalt** [comment]
> 
> 
> Oh, using an alpha version is quite risky 😅
> 
> > If there is a trace or anything else I can grab, please let me know.
> 
> I'm not sure what you like to do. You want to to drag the groups?

No kidding - but it adds to the validity of testing, does it not?

Considering I've been using Fx Nightly for ***years*** and also considering I've been ßeta testing for decades, I'd have thought these types of comments would have, by now, stopped making their rounds at me.

Of course, you don't know me, so, there is that.

**Comment by JohnLGalt** [comment]
> 
> 
> Unfortunately we don't have very good logging set up right now so there's no backtrace you can grab for us.

Ahhh.  Hmmm.

Well, anything I can do on my end to supply you with more info?  I suspect it has to do with some changes in build 71 as this did not occur in previous nightly builds.

**Comment by vaxul** [comment]
> I'd have thought these types of comments would have, by now, stopped making their rounds at me.

I'm sorry. It wasn't meant to be an insult. 
I myself use most of the betas and snapshots.

> Unfortunately we don't have very good logging set up right now so there's no backtrace you can grab for us.

Let us open an separate issue for this topic for discussion.

**Comment by JohnLGalt** [comment]
Works for me.  

I'll look for a new topic, if not, create one.

**Comment by JohnLGalt** [comment]
I just realized I don't have a methodology to reproduce the bug.

Here goes:

I have several defined groups (at least 22, I cannot look right now to verify, PTG is not even opening the group page) in which I have different tabs (except for one pair of groups, I don't have duplicated tabs in any groups, hence the need for grouping in the first place).

My usual methodology is as this:

Keep the very first group that PTG creates as my default to open group, where I open tabs of all sorts (both those I later group as well as those I never group / close after using).

I then have my grouped tabs obviously in other groups.  When I want o change to a group, I click the PTG button at the top of Fx to bring up the panorama view and select the group I want to view, whose tabs I can then view, refresh, or close as I chose.  When done I hit the same button to move to the next group I want to view.

What's happening now is that, after I try to change to a group, one tab of the previous group remains as the first tab, before all of the grouped tabs of the current group.  If I then click on that one tab, all the tabs from the previous group fly open, so I effectively have 2 groups simultaneously showing.  This never occurred before, if I selected a different group, all the tabs in the current group would get hidden as all the tabs in the new group would proliferate on the tab bar).

Just to be sure, I'm going to attempt to load the last tab group backup I have, just in case it is a simple file corruption issue and nothing more (which, at this point, is what I suspect).
